ftplugin/markdown.vim

```diff
@@ -23,9 +23,8 @@ function! MarkdownFold()
   let line = getline(v:lnum)
 
   " Regular headers
-  let depth = match(line, '\(^#\+\)\@<=\( .*$\)\@=')
-  if depth > 0
-    return ">" . depth
+  if (match(line, '^#\+ .*$') == 0)
```

Comment:
Please simplify to the more conventional `line =~# '^#\+ '`.

Reply:
Good point, thanks. It should be better now.